### PR TITLE
shell32 [pl-pl] lang fix

### PR DESCRIPTION
**Pull request by pithwz**
shell32 [pl-pl] lang fix
shell32_pl-pl_fix

### CODE REVIEW

**Comment by binarymaster** [comment]
Cc @reactos/lang-polish

**Comment by Zcooger** [comment]
Yes, it's correct suggestion.

File: dll/win32/shell32/lang/pl-PL.rc [hunk 1]
```diff
@@ -814,35 +814,35 @@ BEGIN
...
     IDS_FILEOOP_FROM "Z %1"
-    IDS_FILEOOP_PREFLIGHT "Przygotowywanie"
+    IDS_FILEOOP_PREFLIGHT "Przygotowywanie operacji"
 
     /* message box strings */
     IDS_RESTART_TITLE "Uruchom ponownie"
-    IDS_RESTART_PROMPT "Czy chcesz ponownie uruchomić system?"
-    IDS_SHUTDOWN_TITLE "Zamknij"
-    IDS_SHUTDOWN_PROMPT "Czy chcesz zamknąć system?"
+    IDS_RESTART_PROMPT "System ReactOS zamyka się i uruchamia ponownie"
```
Comment (archeYR) [inline dll/win32/shell32/lang/pl-PL.rc:826]: Incorrect translation (has different meaning than the english resource). Please revert to previous version.

File: dll/win32/shell32/lang/pl-PL.rc [hunk 2]
```diff
@@ -814,35 +814,35 @@ BEGIN
...
+    IDS_SHUTDOWN_TITLE "Zamknij system"
+    IDS_SHUTDOWN_PROMPT "System ReactOS zamyka się i wyłącza komputer"
```
Comment (archeYR) [inline dll/win32/shell32/lang/pl-PL.rc:828]: Same here.

**Review by archeYR — CHANGES_REQUESTED** [review]

File: dll/win32/shell32/lang/pl-PL.rc (hunk 1 above)
Comment (pithwz) [reply dll/win32/shell32/lang/pl-PL.rc:826]: is correct :)

Comment (archeYR) [reply dll/win32/shell32/lang/pl-PL.rc:826]: No it isn't, unless the english resource is wrong there. If that's a case, then please fix it first and then update translations accordingly.

Comment (binarymaster) [inline dll/win32/shell32/lang/pl-PL.rc:826]:
```suggestion
    IDS_RESTART_PROMPT "Czy chcesz ponownie uruchomić system?"
```

File: dll/win32/shell32/lang/pl-PL.rc (hunk 2 above)
Comment (binarymaster) [inline dll/win32/shell32/lang/pl-PL.rc:828]:
```suggestion
    IDS_SHUTDOWN_PROMPT "Czy chcesz zamknąć system?"
```

**Review by binarymaster — COMMENTED** [review]
Revert incorrect translations.

**Comment by binarymaster** [comment]
I reverted incorrect translations and merged everything else. Feel free to create a separate Pull Request which will fix this problem for all translations, but only if it really a problem.

**Pull request merged**